packages/core/components/Puck/index.tsx

```diff
@@ -318,7 +318,7 @@ export function Puck({
                             display: "grid",
                             padding: 16,
                             gridTemplateAreas: '"left middle right"',
-                            gridTemplateColumns: "288px auto 288px",
```

Comment:
We should have both left and right areas the same width, otherwise the title text won't be entered.

Also suggest we should use the value `344px` instead of `343px` (I've tended to keep things as multiples of 4).

Reply:
oh I see, but can you explain what this (`otherwise the title text won't be entered.`) means? (To recognize context)

- 288px 344px 

<img src="https://github.com/Yuddomack/images/blob/master/%EC%8A%A4%ED%81%AC%EB%A6%B0%EC%83%B7%202023-10-25%20%EC%98%A4%ED%9B%84%205.44.28.png?raw=true" />

- 344px 344px 

<img src="https://github.com/Yuddomack/images/blob/master/%EC%8A%A4%ED%81%AC%EB%A6%B0%EC%83%B7%202023-10-25%20%EC%98%A4%ED%9B%84%205.44.52.png?raw=true" />

If the intention was to make the middle of both sides appear, I think I would need to calculate an additional width equal to the button?!

Reply:
Sorry that was a typo - "entered" should read "centred".

The second scenario looks correct for me. We may wish to look deeper into these styles in the future.

Reply:
ah centered! fix immediately 🔥